This one continues to be flaky. Is that actually related to containerd's shange to use blocksize?;

- https://github.com/moby/moby/issues/47783
- https://github.com/containerd/continuity/pull/163



```
=== Failed
=== FAIL: amd64.integration.system TestDiskUsage/empty (0.00s)
    disk_usage_test.go:43: assertion failed: 
        --- du
        +++ →
          types.DiskUsage{
        - 	LayersSize: 4096,
        + 	LayersSize: 0,
          	Images:     {},
          	Containers: {},
          	... // 3 identical fields
          }
        
    --- FAIL: TestDiskUsage/empty (0.00s)

=== FAIL: amd64.integration.system TestDiskUsage (2.77s)
```

--- a/integration/system/disk_usage_test.go
+++ b/integration/system/disk_usage_test.go
@@ -18,10 +18,6 @@ import (
 func TestDiskUsage(t *testing.T) {
 	skip.If(t, testEnv.DaemonInfo.OSType == "windows") // d.Start fails on Windows with `protocol not available`
 
-	// TODO: If this helps, then fix the root cause.
-	// See: https://github.com/moby/moby/issues/47119
-	// t.Parallel()
-
 	ctx := testutil.StartSpan(baseContext, t)
 
 	d := daemon.New(t)
@@ -40,7 +36,18 @@ func TestDiskUsage(t *testing.T) {
 			next: func(t *testing.T, _ types.DiskUsage) types.DiskUsage {
 				du, err := client.DiskUsage(ctx, types.DiskUsageOptions{})
 				assert.NilError(t, err)
+
+				expectedLayersSize := int64(0)
+				// TODO: Investigate https://github.com/moby/moby/issues/47119
+				// Make 4096 (block size) also a valid value for zero usage.
+				if testEnv.UsingSnapshotter() && testEnv.IsRootless() {
+					if du.LayersSize == 4096 {
+						expectedLayersSize = du.LayersSize
+					}
+				}
+
 				assert.DeepEqual(t, du, types.DiskUsage{
+					LayersSize: expectedLayersSize,
 					Images:     []*image.Summary{},
 					Containers: []*types.Container{},
 					Volumes:    []*volume.Volume{},

